workers-docs/src/content/reference/apis/html-rewriter.md

```diff
@@ -192,9 +196,25 @@ class DocumentHandler {
 
 #### Properties
 
-- `name: String | null`: Read-only, string representing the doctype name. 
+- `name: String | null`: Read-only, string representing the doctype name.
 - `publicId: String | null`: Read only, the quoted string in the doctype after the PUBLIC atom.
-- `systemId: String | null`: Read only, the quoted string in the doctype after the SYSTEM atom or immidiately after the `publicId`. 
+- `systemId: String | null`: Read only, the quoted string in the doctype after the SYSTEM atom or immidiately after the `publicId`.
+
+### End
+
+The `end` function on a document handler allows developers to append content to the end of a document.
+
+```js
+class DocumentHandler {
+  end(end) {
+    // The end of the document
+  }
+}
+```
+
+#### Methods
+
+- `append(content: Content, contentOptions?: ContentOptions): DocumentEnd`: Inserts content after the end of the document.
```

Comment:
what is `DocumentEnd` in this context?

Reply:
It returns itself so you can use it sort of like a builder like the rest of the handler types. Though it's not as useful in this case since you can only append, it's there for consistency sake.

Reply:
ok, makes sense!